index.bs

```diff
@@ -561,6 +574,9 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
     operations implies use of key material managed by the authenticator. Note that for security, [=user verification=] and use
     of [=credential private keys=] must occur within a single logical security boundary defining the [=authenticator=].
 
+    [=User verification=] is a defining characteristic of the [=first-factor use case|first-factor=] and [=single-step use
```

Comment:
I am not sure I understand that.
Why don't we say: The process by which an Authenticator locally authorizes use of the [=credential private key=], for example through a touch, pin code, fingerprint match or other biometric.

Reply:
I think I added this mostly as a way to link UV together with the first-factor use case. Perhaps that's unnecessary.